cert/vault_client.go

```diff
@@ -132,3 +137,14 @@ func (c *vaultClient) keepTokenAlive() {
 		timer.Reset(ttl / 2)
 	}
 }
+
+func getVaultTokenFromFile(c string) string {
+	b, err := ioutil.ReadFile(c) // just pass the file name
+	if err != nil {
+		log.Printf("[WARN] vault: Failed to get Token From File: %s", err)
```

Comment:
nit: `Token From File` should be lowercase. In the debug message below it should say "fetched", not "fetch".

Reply:
Will do